zwave_js_server/model/controller.py

```diff
@@ -9,9 +11,41 @@
     from ..client import Client
 
 
+class InclusionGrantDataType(TypedDict):
+    """Representation of an inclusion grant data dict type."""
+
+    # https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/controller/Inclusion.ts#L48-L56
+    securityClasses: List[SecurityClass]
+    clientSideAuth: bool
+
+
+@dataclass
+class InclusionGrant:
+    """Representation of an inclusion grant."""
+
+    security_classes: List[SecurityClass]
+    client_side_auth: bool
+
+    def to_dict(self) -> InclusionGrantDataType:
+        """Return InclusionGrantDataType dict from self."""
+        return {
+            "securityClasses": self.security_classes,
+            "clientSideAuth": self.client_side_auth,
+        }
+
+    @staticmethod
```

Comment:
Make this a classmethod.
```suggestion
    @classmethod
```